core/src/main/java/fr/sncf/osrd/train/RollingStock.java

```diff
@@ -61,17 +70,43 @@
      */
     public final TractiveEffortPoint[] tractiveEffortCurve;
 
+    @Override
+    public double getMass() {
+        return mass;
+    }
+
+    @Override
+    public double getInertia() {
+        return inertia;
+    }
+
+    @Override
+    public double getLength() {
+        return length;
+    }
+
+    @Override
     /**
      * Gets the rolling resistance at a given speed, which is a force that always goes
      * opposite to the train's movement direction
      */
-    public double rollingResistance(double speed) {
+    public double getRollingResistance(double speed) {
```

Comment:
Delete that one because it is redundant with getRollingResistance in TrainPhysicsIntegrator